volontulo/tests/views/test_auth.py

```diff
@@ -3,11 +3,13 @@
 u"""
 .. module:: test_auth
 """
-
 from django.contrib.auth.models import User
 from django.test import Client
+from django.test import TestCase
 from django.test import TransactionTestCase
 
+from volontulo.tests.views.test_usersprofile import TestUsersProfile
```

Comment:
I looks that it will not work after PR #218 as You have moved common methods into common.py